src/main/thrift/parquet.thrift

```diff
@@ -576,5 +607,14 @@ struct FileMetaData {
    * e.g. impala version 1.0 (build 6cf94d29b2b7115df4de2c06e2ab4326d721eb55)
    **/
   6: optional string created_by
+
+  /**
+   * Sort order used for each column in this file.
+   *
+   * If this list is not present, then the order for each column is assumed to
+   * be Signed. In addition, min and max values for INTERVAL or DECIMAL stored
```

Comment:
If the order isn't present, then it was byte-wise signed comparsion that is useless. Readers should ignore the values.